### ISSUE TITLE
Polarizable water in OpenMM

### ISSUE THREAD

**Issue by rmatsum836** [mosdef-hub/mosdef-workflows#8]
It was mentioned in the meeting the other day that it would be nice to have various water models.  I have a OpenMM workflow for some polarizable models that I think would be a good addition. 

Question is, should this exist as its own repo, or should the `water-box` workflow be expanded upon?

**Comment by ahy3nz** [comment]
I think this could be a helpful addition of polarizable models. In my opinion, I don't think it has to be a particularly exotic example like a pore, but just showing what a polarizable XML looks like, how to work with a mb-compound-that-becomes-polarizable?

**Comment by rmatsum836** [comment]
Yeah this would be just a box of water.

**Comment by mattwthompson** [comment]
> Question is, should this exist as its own repo, or should the water-box workflow be expanded upon?

imo a separate example

**Comment by justinGilmer** [comment]
Yeah I think a separate example would be more appropriate. Maybe we should rename the water box examples. `water-box-spce`
`water-box-polarizable`?
